### PR TITLE
Initial Schema

### PR DESCRIPTION
**Pull request by Ian-Stewart-Binks**
The pull request creates the initial SVG schema. The schema will be extended to be able to manage other data as necessary (for instance: classes).

Any suggestions are welcome, as always.

### CODE REVIEW

File: hs/database/Tables.hs [hunk 1]
```diff
@@ -65,6 +65,32 @@ Distribution
...
+    height Rational
+    xPos Rational
+    yPos Rational
+    colour String
+    deriving Show
+
+Texts
+    gId Int
+    rId Int
+    yPos Rational
```
Comment (david-yz-liu) [inline hs/database/Tables.hs]: Hmmm. Text which belongs to a rect should be centred inside the rect.

Text that doesn't belong to a rect (region labels?) need to have both x and y positions.

All three of these attributes (rId, yPos, xPos) should be Maybes.

File: hs/database/Tables.hs [hunk 2]
```diff
@@ -65,6 +65,32 @@ Distribution
...
+    text String
+
+Paths
+    gId Int
+    d String
```
Comment (david-yz-liu) [inline hs/database/Tables.hs]: What I'd like to see for Paths is a list of points `[(Rational, Rational)]`.

Comment (Ian-Stewart-Binks) [inline hs/database/Tables.hs]: Done.

File: hs/database/Tables.hs (hunk 1 above)
Comment (Ian-Stewart-Binks) [inline hs/database/Tables.hs]: Text which belongs to a rect should not always be centered inside the rect.

Comment (Ian-Stewart-Binks) [inline hs/database/Tables.hs]: Change has been made, as well.

**Comment by david-yz-liu** [comment]
Thanks for reminding me about the text positioning; I really hope we come up with something better some day.

If this is the case, it makes sense to make the `xPos` and `yPos` just Rationals, and not Maybes.

**Comment by Ian-Stewart-Binks** [comment]
It may be best to impose some sort of line height on all nodes, or some sort of uniform formatting for nodes with multi-line text (eventually).

**Comment by david-yz-liu** [comment]
We can do this when putting data into the database.

**Comment by Ian-Stewart-Binks** [comment]
Changed `xPos` and `yPos` back to `Rational`s.

**Comment by david-yz-liu** [comment]
Thanks, @Ian-Stewart-Binks!

**Pull request merged**